modules/virt-about-changing-removing-mediated-devices.adoc

```diff
@@ -3,21 +3,18 @@
 // * virt/virtual_machines/advanced_vm_management/virt-configuring-mediated-devices.adoc
 
 :_content-type: CONCEPT
-
 [id="about-changing-removing-mediated-devices_{context}"]
 = About changing and removing mediated devices
 
-The cluster's mediated device configuration can be updated with {VirtProductName} by:
+You can reconfigure or remove mediated devices in several ways:
 
-* Editing the `HyperConverged` CR and change the contents of the `mediatedDevicesTypes` stanza.
+* Edit the `HyperConverged` CR and change the contents of the `mediatedDevicesTypes` stanza.
```

Comment:
Also due to the bug [1] being fixed in 4.14, we need to update all instances of `mediatedDevicesTypes` to `mediatedDeviceTypes`.  

Note `Device` singular and `Devices` plural above.

[1] - https://bugzilla.redhat.com/show_bug.cgi?id=2054863#c10